p2p/ipld/plugin/nodes/nodes.go

```diff
@@ -13,8 +13,7 @@ import (
 	"github.com/ipfs/go-cid"
 	"github.com/ipfs/go-ipfs/core/coredag"
 	"github.com/ipfs/go-ipfs/plugin"
-	format "github.com/ipfs/go-ipld-format"
-	node "github.com/ipfs/go-ipld-format"
```

Comment:
😆

Reply:
I know ... A bit embarrassing 😬 Glad you found out that we weren't running CI on this 👍🏼